anscombe.py/main.py

```diff
@@ -0,0 +1,84 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+import pandas as pd
+import seaborn as sns
+
+
```

Comment:
Please add any welcoming message to indicate what is happening in the program while launched from the console